core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -912,8 +927,10 @@ class GroupCoordinator(val brokerId: Int,
   def onExpireHeartbeat(group: GroupMetadata, memberId: String, isPending: Boolean, heartbeatDeadline: Long) {
     group.inLock {
       if (isPending) {
-        debug(s"Pending member $memberId has been removed after session timeout expiration.")
```

Comment:
Out of curiosity, why did we remove this?

Reply:
Yeah, I think the message is useful. I'd even consider raising the verbosity to info. My only comment was to add groupId to it.

Reply:
+1 to having this as `info`.

Reply:
ok cool. I thought this was adding too much verbosity. I did find it useful when debugging (but this was my first time looking at these logs). I'll add a similar line in `handleLeaveGroup`.

Reply:
message added back with info level and with groupID